suite-native/atoms/src/Button/Button.tsx

```diff
@@ -19,10 +19,12 @@ export type ButtonColorScheme =
     | 'secondary'
     | 'tertiaryElevation0'
     | 'tertiaryElevation1'
+    | 'dangerBold'
     | 'dangerElevation0'
-    | 'dangerElevation1'
```

Comment:
Why do we have now tertiaryElevation 0 and 1, but warning and danger have only elevation 0 + bold? The naming does not make sense to me if there is only one level of elevation. Should not all the variants have consistent naming?

Reply:
Names are based on names of used background colors. `dangerElevation1` and `warningElevation1` were used only once so I removed those, since we dont have those in the color scheme anymore. 
 
I didnt want to remove `tertiaryElevation1` and `tertiaryElevation0` which is used on 50+ places because it would mean changing current appearances. It felt to be much bigger change that would need UI/UX review. 

I believe we can refactor it later and I didnt want to widen the scope of this task even more since it is about adding the new colors. If we want to resolve this, I suggest to create a new task, involve Sheny and if possible remove the tertiary completely since those are not in the buttom color scheme list anymore: 

https://www.figma.com/file/Y2E8dcdNYrSMGIVRjYFXKp/%5BSuite%5D-Components?type=design&node-id=1-1238&mode=design&t=P7LS3nQkikJjVjdY-0

But I suggest not to have it as a part of this PR.

Reply:
If you feel, there is a better naming convention suitable, I am open to any suggestions :). I just followed current one

Reply:
I should probably rename danger to red, warning to yellow and info to blue though, to follow fully the naming convention. That names i took from button type names, but does not fit. Good point.

Reply:
updated in [091dbdb](https://github.com/trezor/trezor-suite/pull/12088/commits/091dbdb3257b0849b013830e97ff5defaaa7eedc)

Reply:
@shenkys are you aware of this? We used different elevations across the whole app. If you would use the button on the lowest level of the screen directly, you would use elevation0. In case that the button would be inside of some "elevated" element (e.g. Card) you would use elevation 1 which has a little bit different shade.

I am not against deleting this, just want to make sure we don't delete a feature of styling that we will want to use in the future.

Reply:
I was not aware of this until now. If it worked like that, we should keep it. Let's discuss it together on call.

Reply:
Adding back red and yellow elevation1

48e23410af